packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -376,6 +379,12 @@ export interface RepositoryProps {
    * Options to add additional security groups to the Repository.
    */
   readonly securityGroupsOptions?: RepositorySecurityGroupsOptions;
+
+  /**
+   * The Deadline Repository settings file to import.
+   * @see https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/repository-settings-importer-exporter.html
+   */
+  readonly repositorySettings?: Asset;
```

Comment:
We should describe the default behaviour here too.

Reply:
Added